src/extension.ts

```diff
@@ -86,11 +88,11 @@ export async function activate(context: vscode.ExtensionContext): Promise<Extens
         vscode.commands.registerCommand('docs.build', (uri) => {
             buildController.build(getCorrelationId(), uri, credentialController.credential);
         }),
-        vscode.commands.registerCommand('docs.openPage', (uri: vscode.Uri) => {
-            vscode.env.openExternal(uri);
+        vscode.commands.registerCommand('learnMore', (code: string) => {
```

Comment:
what code is this?

Reply:
the errorCode of the diagnostic

Reply:
the errorCode of the diagnostic

---
In reply to: [382361181](https://github.com/docascode/vscode-docs-build/pull/45#discussion_r382361181) [](ancestors = 382361181)

Reply:
I see, thanks